### PR TITLE
Submit Blobert Showdown

### PR DESCRIPTION
**Pull request by 0xGavinOwen**

### CODE REVIEW

**Comment by tarrencev** [comment]
Fun idea! Is there a link to a playable version of this?

File: submissions/TEMPLATE.md [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
-To submit your project:
-- Copy / Paste this template into a file with your project names as the file name, using `shotgun-case`.
-- Replace the placeholder content below with your projects info
-- Remove this overview section
-- Create a pull request to the `dojoengine/game-jams` repo with your submission
-
----
-
-# Project Name
+# Blobert Showdown
```
Comment (tarrencev) [inline submissions/TEMPLATE.md:1]: Please revert the changes to the TEMPLATE file

**Comment by 0xGavinOwen** [comment]
> Fun idea! Is there a link to a playable version of this?

**Comment by 0xGavinOwen** [comment]
> Fun idea! Is there a link to a playable version of this?
Honestly speaking,  we are not able to finish it in time. But since this project is important for our overall roadmap we will still continue to finish it in the next 2 weeks. In the mean time, our readme shows the overall game overview and demo.
